sighmon/co2_sensor_scd4x_esp32_http_server
# #2 Add SCD-30 compatibility

Resolves #2 

- [x] Add compatibility with the [Adafruit SCD-30](https://www.adafruit.com/product/4867)

## Testing

1. Connect your SCD-30 sensor blue SDA to pin `21` on the ESP32
1. Connect your SCD-30 sensor yellow SCL to pin `22` on the ESP32
1. Connect your SCD-30 red to `3.3v` on the ESP32
1. Connect your SCD-30 black to `ground` on the ESP32
1. Upload the Arduino sketch, and `curl 192.168.1.104` to see the readings
